.rubocop_todo.yml

```diff
@@ -310,3 +310,14 @@ Layout/LineLength:
     - 'spec/system/planning_applications/validating_spec.rb'
     - 'db/migrate/20211020153256_create_delayed_jobs.rb'
     - 'db/migrate/20211122103943_update_enquiries_paragraph.rb'
+
+Style/OpenStructUse:
```

Comment:
Hmm. I think this is okay, but let's find time to remove all the `OpenStructs` at some point!

I have a personal wish-list of refactors for between now and January 😆